test/rpc/test_rpc_misc.py

```diff
@@ -193,13 +183,66 @@ def test_get_events_wrong_blockid_type():
     """
     Test RPC get_events with malformed request.
     """
+    params = create_get_events_filter()
+    params["filter"]["from_block"] = {"block_number": "0x0"}
     resp = rpc_call(
         "starknet_getEvents",
-        params=create_get_events_rpc(BLOCK_FROM_0_TO_LATEST_WRONG_BLOCK_TYPE)["params"],
+        params=params,
     )
     assert resp["error"]["code"] == PredefinedRpcErrorCode.INVALID_PARAMS.value
 
 
+@devnet_in_background(*PREDEPLOY_ACCOUNT_CLI_ARGS)
+def test_get_events_continuation_token():
+    """
+    Test RPC get_events returning continuation token.
+    """
+    deploy_info = deploy(EVENTS_CONTRACT_PATH)
+    total_invokes = 3
+    for i in range(total_invokes):
+        invoke(
+            calls=[(deploy_info["address"], "increase_balance", [i])],
+            account_address=PREDEPLOYED_ACCOUNT_ADDRESS,
+            private_key=PREDEPLOYED_ACCOUNT_PRIVATE_KEY,
+        )
```

Comment:
How about making it a separate fixture defined in this file?

Reply:
Not sure how to do this. Is it possible to run devnet on a fixture, like with `@devnet_in_background(*PREDEPLOY_ACCOUNT_CLI_ARGS)`? I marked the fixture with it and marked `test_get_events_continuation_token` with `@pytest.mark.usefixtures("invoke_increase_balance")` - the test then gives an error 
```
E           requests.exceptions.ConnectionError: HTTPConnectionPool(host='127.0.0.1', port=49990): 
Max retries exceeded with url: /gateway/add_transaction (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x119fe99d0>:
 Failed to establish a new connection: [Errno 61] Connection refused'))
```
@FabijanC

Reply:
Maybe Artur was refering to something like the existing "deploy_info" and "invoke_info" in `test/rpc/conftest.py` where you could add transactions that you need.

But IMO it's ok to keep the invokes explicitly here in the body of the test.